src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Dto/RemoteVettingProcessDto.php

```diff
@@ -41,15 +41,17 @@ class RemoteVettingProcessDto implements Serializable
      * @var AttributeListDto
      */
     private $attributes;
+    private $identityProviderName;
 
     /**
      * @param ProcessId $processId
      * @param RemoteVettingTokenDto $token
+     * @param string $identityProviderName
      * @return RemoteVettingProcessDto
      */
-    public static function create(ProcessId $processId, RemoteVettingTokenDto $token)
+    public static function create(ProcessId $processId, RemoteVettingTokenDto $token, $identityProviderName)
     {
-        return new self($processId, $token, new RemoteVettingStateInitialised(), AttributeListDto::notSet());
+        return new self($processId, $token, new RemoteVettingStateInitialised(), AttributeListDto::notSet(), $identityProviderName);
```

Comment:
Curious PHPCS did not trigger on this too long loc. Please reorganize and turn it into a multiline statement.

Reply:
It's valid according to the used ruleset in phpcs.xml.